src/main/java/datawave/microservice/config/metrics/DatawaveMetricsConfig.java

```diff
@@ -29,12 +30,16 @@ public DatawaveMetricsConfig(MetricsConfigurationProperties configProps) {
     @Override
     public void configureReporters(MetricRegistry metricRegistry) {
         try {
-            MetricsReporterFactory factory = configProps.getFactoryClass().newInstance();
+            MetricsReporterFactory factory = configProps.getFactoryClass().getDeclaredConstructor().newInstance();
             ScheduledReporter reporter = factory.forRegistry(metricRegistry).convertRatesTo(TimeUnit.SECONDS).convertDurationsTo(TimeUnit.MILLISECONDS)
                             .build(configProps.getHost(), configProps.getPort());
             registerReporter(reporter).start(configProps.getInterval(), configProps.getIntervalUnit());
         } catch (InstantiationException | IllegalAccessException e) {
             throw new RuntimeException("Unable to instantiate metrics reporter factory class " + configProps.getFactoryClass() + ": " + e.getMessage(), e);
+        } catch (InvocationTargetException e) {
```

Comment:
I think you could have simply added these addition exceptions to the previous catch clause.